File: SDKs/openlinksw.com/osdb/1.0.0/java/lib/src/main/java/openapisdk/models/shared/ListServicesResponse.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

public class ListServicesResponse {
    @JsonProperty("api")
    public String api;
    public ListServicesResponse withApi(String api) {
        this.api = api;
        return this;
    }
    @JsonProperty("method")
    public String method;
    public ListServicesResponse withMethod(String method) {
        this.method = method;
        return this;
    }
    @JsonProperty("response")
    public ServiceDescription[] response;
    public ListServicesResponse withResponse(ServiceDescription[] response) {
        this.response = response;
        return this;
    }
    @JsonProperty("status")
    public ListServicesResponseStatusEnum status;
    public ListServicesResponse withStatus(ListServicesResponseStatusEnum status) {
        this.status = status;
        return this;
    }
}

File: SDKs/openlinksw.com/osdb/1.0.0/python/src/sdk/models/shared/listservicesresponse.py
import dataclasses
from enum import Enum
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import servicedescription as shared_servicedescription

class ListServicesResponseStatusEnum(str, Enum):
    SUCCESS = "success"


@dataclass_json
@dataclasses.dataclass
class ListServicesResponse:
    api: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('api') }})
    method: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('method') }})
    response: list[shared_servicedescription.ServiceDescription] = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('response') }})
    status: ListServicesResponseStatusEnum = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('status') }})
    
